packages/datatrak-web/src/api/mutations/submitSurvey/useSubmitSurvey.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Tupaia
```

Comment:
I split up this file as it was getting pretty big and doing a few different things